docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -11,19 +11,21 @@
    - To check your Windows version, run `winver` on the command line.
 
 ## Software prerequisites
+For specific versions we use for each release, see [install-windows-prereqs.ps1](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1#L6-L36).
 - [Microsoft Visual Studio Build Tools 2019](https://aka.ms/vs/15/release/vs_buildtools.exe)
 - [Git for Windows 64-bit](https://git-scm.com/download/win)
-- [Clang/LLVM for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe)
+- [Clang/LLVM 8.0.1 for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe)
```

Comment:
I thought we're using clang 10 already, no?

Reply:
Thanks Ming-Wei. Seems like this doc was missed in the Clang 10 upgrade

Reply:
Changed name and link to Clang 10.0.0